### PR TITLE
OEL-579: Medley branch with all upgrade to BCL 0.9.0 

### PR DESCRIPTION
**Pull request by escuriola**
Jira issue(s):
- https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-579

Contains: 
- OEL-474: All pattern tests with BCL 0.9.0 (https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-474)
- OEL-424/FRONT-2488: select multiple with slimSelect (https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-424)
- OEL-583/FRONT-2491: Toast (https://citnet.tech.ec.europa.eu/CITnet/jira/browse/OEL-583)

### CODE REVIEW

File: templates/patterns/navbar/pattern-navbar.html.twig [hunk 1]
```diff
@@ -44,21 +45,41 @@
...
+      </a>
+    {% elseif navbar_branding is not empty %}
       {{ navbar_branding }}
     {% endif %}
+    {% if not _disable_collapse %}
     <button
-      class="navbar-toggler"
-      type="button"
-      data-bs-toggle="collapse"
+      class='navbar-toggler'
```
Comment (drishu) [inline templates/patterns/navbar/pattern-navbar.html.twig]: what's the change to single quotes for ?

File: templates/patterns/toasts/pattern-toasts.html.twig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{#
```
Comment (drishu) [inline templates/patterns/toasts/pattern-toasts.html.twig]: pls also implement the feedback I left in the toast PR

**Review by drishu — CHANGES_REQUESTED** [review]

File: tests/fixtures/markup_rendering.yml [hunk 3]
```diff
@@ -17,9 +17,47 @@ textarea:
...
+  assert:
+    equals:
+      'label.form-label': "Select"
+      'option[value=0]': "One"
+      'option[value=1]': "Two"
+      'option[value=2]': "Three"
+    count:
+      'label.form-label': 1
+      'select.form-select': 1
+submit:
```
Comment (drishu) [inline tests/fixtures/markup_rendering.yml]: lets do more variants for all these, example here in submit we can:
check for btn-primary, btn-secondary, and a disabled one

**Review by drishu — CHANGES_REQUESTED** [review]

File: tests/fixtures/markup_rendering.yml (hunk 3 above)
Comment (escuriola) [reply tests/fixtures/markup_rendering.yml]: This part is done. I did tests for each pattern with all variants.

File: templates/patterns/toasts/pattern-toasts.html.twig (hunk 2 above)
Comment (drishu) [reply templates/patterns/toasts/pattern-toasts.html.twig]: we postpone till FRONT issue is resolved

**Review by drishu — APPROVED** [review]

**Review by abel-santos-corral — APPROVED** [review]

**Pull request merged**